src/main/java/org/broadinstitute/hellbender/utils/IntervalUtils.java

```diff
@@ -59,6 +59,28 @@
 
     private static final Logger logger = LogManager.getLogger(IntervalUtils.class);
 
+    /**
+     * Compare two locations using a {@link htsjdk.samtools.SAMSequenceDictionary} sequence ordering
```

Comment:
Add javadoc:

`@return 0 if first and second are equal, a negative value if first < second, or a positive value if first > second, using the ordering in the provided dictionary`